src/Diagnostics/FxCop/System.Runtime.Analyzers/Test/Design/DefineAccessorsForAttributeArgumentsTests.Fixer.cs

```diff
@@ -309,12 +310,12 @@ Inherits Attribute
     Public Sub New(name As String)
         m_name = name
     End Sub
-    
+
     Public Property Name() As String
         Friend Get
             Return m_name
         End Get
-        Set
+        Friend Set
```

Comment:
[minor] This code is not valid VB (it produces another error - 'BC31101	Access modifier can only be applied to either 'Get' or 'Set', but not both.') Consider adding scenario / test that is valid C# and only produces CA1019.